.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: scala
 
 scala:
-  - 2.12.1
+  - 2.12.6
 
 env:
-  - JDK=oraclejdk8
+  global:
+    - JDK=oraclejdk8
+    - AWS_REGION=us-west-2
 
 before_script:
   - jdk_switcher use $JDK
 
-script: sbt ++$TRAVIS_SCALA_VERSION clean stack/clean 'test-only -- timefactor 10' 'stack/test-only -- timefactor 10'
+script: sbt ++$TRAVIS_SCALA_VERSION clean stack/clean 'testOnly -- timefactor 10' 'stack/testOnly -- timefactor 10'
```

Comment:
I think the `README` needs to also be updated to reflect the change from `test-only` to `testOnly`.

Reply:
I get an error when trying to run the integration tests:
```
[error] /Users/coliver/Workspace/Github/cloudflare-public-hostname-lambda/stack/src/it/scala/com/dwolla/cloudformation/cloudflare/StackIntegrationSpec.scala:5:78: Unused import
[error] import com.amazonaws.services.cloudformation.model.{ValidateTemplateRequest, ValidateTemplateResult}
[error]                                                                              ^

```

Reply:
Addressed in d9e2b17